estate/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, fields
+
+class EstateProperty(models.model):
+    _name = "estate_property"
+    _description = "Table that stores the estate properties"
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+
+    date_availability = fields.Date(string="Available From")
+    expected_price = fields.Float(string="Expected price", required=True)
+    selling_price = fields.Float(string="Selling price")
+    bedrooms = fields.Integer(string="Bedrooms")
+    living_area = fields.Integer(string="Living Arez (sqm)")
+    facades = fields.Integer(string="Facades")
+    garage = fields.Boolean(string="Garage")
+    garden = fields.Boolean(string="Garden")
```

Comment:
```suggestion
    description = fields.Text()
    postcode = fields.Char()

    date_availability = fields.Date(string="Available From")
    expected_price = fields.Float(required=True)
    selling_price = fields.Float()
    bedrooms = fields.Integer()
    living_area = fields.Integer(string="Living Area (sqm)")
    facades = fields.Integer()
    garage = fields.Boolean()
    garden = fields.Boolean()
```
No need for the strings here, they are automatically generated from the variable name